build_test.go

```diff
@@ -215,8 +195,9 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 
 	context("when build plan entries require pipenv at build/launch", func() {
 		it.Before(func() {
-			entryResolver.MergeLayerTypesCall.Returns.Build = true
-			entryResolver.MergeLayerTypesCall.Returns.Launch = true
+			buildContext.Plan.Entries[0].Metadata = make(map[string]interface{})
```

Comment:
This can be inlined like in https://github.com/paketo-buildpacks/pip/pull/256#discussion_r868058514